plugins/repository-azure/src/main/java/org/opensearch/repositories/azure/AzureStorageService.java

```diff
@@ -159,10 +162,27 @@ public Tuple<BlobServiceClient, Supplier<Context>> client(String clientName, BiC
         return new Tuple<>(state.getClient(), () -> buildOperationContext(azureStorageSettings));
     }
 
+    private StorageEndpoint getStorageBlobEndpoint(final AzureStorageSettings settings) {
+        try {
+            String endpointSuffix = settings.getEndpointSuffix();
+            if (!Strings.hasText(endpointSuffix)) {
+                endpointSuffix = "core.windows.net";
```

Comment:
Please use Azure constants (com.azure.storage.common.implementation.Constants):

```suggestion
                endpointSuffix = Constants.DEFAULT_DNS; 
```

Reply:
interestingly `Constants.DEFAULT_DNS` does not exist, but I managed to use `Constants.ConnectionStringConstants.DEFAULT_DNS` instead, it has the value "core.windows.net" too.

Reply:
Thank you.